### PR TITLE
Krav Maga Nerfs

### PR DESCRIPTION
**Pull request by Mqiib**
# Document the changes in your pull request

For starters, the leg seep is no longer a hard 4 second stun because that's just stun baton hands with one extra step (clicking a button). For a martial art that is FREE for the warden with no drawbacks besides not wearing insulated gloves, its a bit much. Also the lung punch has had it's losebreath stacks reduced by 40% because without any kind of oxy healing chems two hits would knock you out for a while. Should be a bit less scary powerful.

# Wiki Documentation

Lung punch only applies 3 losebreath stacks, up to a max of 6.

# Changelog

:cl:  

tweak: Krav maga lung punch losebreath reduced by 40%
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
Based

**Comment by TheRyeGuyWhoWillNowDie** [comment]
maybe make warden's krav hands use a snowflake nerfed type of krav maga while keeping the syndie gloves the way they are

**Comment by Marmio64** [comment]
yeah this'll make the syndi gloves pretty bad to use

File: code/datums/martial/krav_maga.dm [hunk 1]
```diff
@@ -93,7 +93,7 @@
 					  	span_userdanger("[A] leg sweeps you!"))
 	playsound(get_turf(A), 'sound/effects/hit_kick.ogg', 50, 1, -1)
 	D.apply_damage(A.dna.species.punchdamagehigh / 2, BRUTE)	//5 damage
-	D.Paralyze(40)
+	D.Knockdown(40)
```
Comment (JamieD1) [inline code/datums/martial/krav_maga.dm]:
```suggestion
	D.Paralyze(40)
```

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**